README.adoc

```diff
@@ -59,28 +78,87 @@ module "backup" {
     iam_role_arn = module.iam_assumable_role_velero.iam_role_arn
   }
 
-  dependency_ids = {
-    kube-prometheus-stack = module.kube-prometheus-stack.id,
-    argocd                = module.argocd_bootstrap.id # TODO Can cause issues on the first cluster bootstrap. If it's the case, deploy after the last argocd.
-  }
```

Comment:
Did you intend to remove the `dependency_ids`?

Reply:
I originally thought that it wasn't really part of the module usage, but more like a deployment config. But now I agree that it should also be documented in this example.